### PR TITLE
Hotfix/aa bv 102

### PR DESCRIPTION
**Pull request by drew2a**
Guys, please look at CONTRIBUTING file.

### CODE REVIEW

**Comment by drew2a** [comment]
If you see any typos or other mistakes, please say it to me.

File: CONTRIBUTING [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change.
+
+Please note we have a [code of conduct](CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
+
+## Branching
+We follow  [git-flow branching model](http://nvie.com/posts/a-successful-git-branching-model/).
```
Comment (sofabeat) [inline CONTRIBUTING]: This one seems easier to understand: https://danielkummer.github.io/git-flow-cheatsheet/

File: CONTRIBUTING [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* hotfix/
+* release/
+
+We suggest you to use [git-flow extensions](https://github.com/nvie/gitflow).
+
+## Code style
+We follow Google code style guides:
+* JavaScript: https://google.github.io/styleguide/jsguide.html
+* TypeSript: https://github.com/google/ts-style
+
```
Comment (sofabeat) [inline CONTRIBUTING]: JSON style guide https://google.github.io/styleguide/jsoncstyleguide.xml

**Comment by drew2a** [comment]
@sofabeat I have added fixes based on your comments

File: CONTRIBUTING (hunk 1 above)
Comment (drew2a) [reply CONTRIBUTING]: added

File: CONTRIBUTING (hunk 2 above)
Comment (drew2a) [reply CONTRIBUTING]: added

**Pull request merged**